aws/tf/modules/sra/network.tf

```diff
@@ -7,7 +7,7 @@ module "vpc" {
 
   count = var.operation_mode != "custom" ? 1 : 0
 
-  name = "${var.resource_prefix}-data-plane-VPC"
+  name = "${var.resource_prefix}-classic-compute-plane-vpc"
```

Comment:
It feels strange to rename one part of networking from data-plane -> classic-compute-plane, but to rename the rest as data-plane -> workspace

Just seems odd to go from 1 name for the data plane to referring to it both as a classic computer plane and also a "workspace"

Reply:
Super minor comment, just thinking out loud.

Reply:
no it's fair - the reason I renamed the data plane SG to 'workspace' is that the privatelink sg is 'privatelink', so those are technically both part of the classic compute plane, I thought I'd add another level of granularity for them

Reply:
Makes a ton of sense to me